src/lakefs_spec/client_helpers.py

```diff
@@ -63,3 +63,8 @@ def revert(client: LakeFSClient, repository: str, branch: str, parent_number: in
     client.branches_api.revert_branch(
         repository=repository, branch=branch, revert_creation=revert_creation
     )
+
+
+def rev_parse(client: LakeFSClient, repository: str, ref: str, parent: int = 0) -> str:
+    revisions = client.refs_api.log_commits(repository=repository, ref=ref).results
+    return revisions[parent].id
```

Comment:
This is unsafe, and should be guarded by a check like `if not revisions`.

Reply:
Added